site/content/en/docs/Guides/Client SDKs/local.md

```diff
@@ -149,3 +172,52 @@ wget https://raw.githubusercontent.com/googleforgames/agones/{{< release-branch
 chmod o+r gameserver.yaml
 docker run --network=host --rm -v $(pwd)/gameserver.yaml:/tmp/gameserver.yaml us-docker.pkg.dev/agones-images/release/agones-sdk:{{<release-version>}} --local -f /tmp/gameserver.yaml
 ```
+
```

Comment:
The same as our other docs: https://github.com/googleforgames/agones/blob/3d57eaa4fe18590634c17137e369302c981b91b9/site/content/en/docs/Integration%20Patterns/allocation-from-fleet.md?plain=1#L28

```suggestion

## Next Steps:
* Learn how to connect your local development game server binary into a running Agones Kubernetes cluster for even more live [development options](({{< ref "/docs/Guides/local-game-server.md" >}})).

```

(This would be at the bottom of the page -- see note on moving below to `local-game-server.md`).

Reply:
Added/updated `Next Steps` on `site/content/en/docs/Guides/Client SDKs/local.md` and `site/content/en/docs/Guides/local-game-server.md` to link to the new `site/content/en/docs/Advanced/out-of-cluster-dev-server.md` page.